### DISCUSSION TITLE
Instruction tuning GPT-2 video, github code and dataset

### DISCUSSION THREAD

**Discussion by cpgrant** [karpathy/build-nanogpt#15] [Ideas]
Keep up the AI for good work. Would like to suggest:
- a video about instruction tuning gpt-2 including github code and dataset
- a video about finetuning gpt-2 with github code + domain specific dataset

Watched and ran your train_gpt2.py from the build-nanogpt repository on a desktop 4090 GPU.
It ran perfectly after adjusting a few variables suggested by ChatGPT. Initially it ran into a memory error. Thank you 😊

**Comment by dustinwloring1988** [comment]
If you wanted to train on github code would it just be better to add it to your pretraining?

**Comment by ViswanathaReddyGajjala** [comment]
I think @cpgrant  might be interested in understanding and learning about instruction tuning in general. This knowledge can help in building private applications.
